src/docfx/build/context/BuildScope.cs

```diff
@@ -251,6 +252,11 @@ private static (Func<string, bool>, FileMappingConfig)[] CreateGlobs(Config conf
                         return JsonUtility.ReadMime(reader, filePath);
                     }
                 case FileFormat.Yaml:
+                    if (filePath.Origin == FileOrigin.Generated)
```

Comment:
hmm, `Input` seems to be a more appropriate place to put the whole `ReadMime` method

Reply:
Fixed.